package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rhumbix/tcomb-json-schema",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

Comment:
did we make a new package for this? The [repo still shows it as 1.0.9](https://github.com/Rhumbix/tcomb-json-schema/blob/master/package.json#L3) and shows the older version of lodash.